agent/api/serviceconnect/service_connect.go

```diff
@@ -32,6 +32,8 @@ type RuntimeConfig struct {
 	StatsRequest string `json:"statsRequest"`
 	// HTTP Path + Params to drain ServiceConnect connections
 	DrainRequest string `json:"drainRequest"`
+	// SC pause container IP address - used for bridge-mode CNI configuration
+	PauseContainerIPConfig *PauseContainerIPConfig `json:"pauseContainerIPConfig,omitempty"`
```

Comment:
Would you prefer storing these as `NetworkConfig` along with pause container associated with customer containers ?

Reply:
Do you mean storing it directly inside `ecscni.ServiceConnectConfig` [struct](https://github.com/aws/amazon-ecs-agent/blob/feature/service-connect/agent/ecscni/types_linux.go#L170)? We collect AppNet pause IPs from docker daemon, but the method that we create this config object currently does not have a handle of docker client. I'd rather keep it that way, because this config object is intended to be an API object for CNI plugin, so it probably shouldn't need to know or care about the container runtime.